HiP-DataStore/Core/WriteModel/TagIndex.cs

```diff
@@ -0,0 +1,44 @@
+﻿using PaderbornUniversity.SILab.Hip.DataStore.Model.Events;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Core.WriteModel
+{
+    public class TagIndex : IDomainIndex
+    {
+        private readonly Dictionary<int, TagInfo> _tag = new Dictionary<int, TagInfo>();
```

Comment:
What do we need the event IDs for? I think a HashSet<string> should suffice, no need for a dictionary. Event IDs are unique anyway, so entries in the dictionary will never be updated or deleted.